{
	"title": "CVE-2021-20226",
	"cve": "CVE-2021-20226",
	"notes": {
		"description": "A use-after-free flaw was found in the io_uring in Linux kernel, where a local attacker with a user privilege could cause a denial of service problem on the system The issue results from the lack of validating the existence of an object prior to performing operations on the object by not incrementing the file reference counter while in use. The highest threat from this vulnerability is to data integrity, confidentiality and system availability.",
		"published": "2021-02-23",
		"modified": "2021-04-01"
	},
	"references": [
		{
			"url": "https://security.netapp.com/advisory/ntap-20210401-0001/",
			"description": "CONFIRM:https://security.netapp.com/advisory/ntap-20210401-0001/"
		},
		{
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1873476",
			"description": "MISC:https://bugzilla.redhat.com/show_bug.cgi?id=1873476"
		}
	]
}
